src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -168,4 +169,28 @@ public static OwedAmount parseOwedAmount(String owedAmount) throws ParseExceptio
         }
         return new OwedAmount(trimmedOwedAmount);
     }
+
+    /**
+     * Parses a {@code String hoursPaid} into a {@code double}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the {@code hoursPaid} is invalid.
+     */
+    public static double parseHoursPaid(String hoursPaid) throws ParseException {
+        requireNonNull(hoursPaid);
+        String trimmedHoursPaid = hoursPaid.trim();
+        double hour;
+
+        try {
+            hour = Double.parseDouble(trimmedHoursPaid);
+        } catch (NumberFormatException nfe) {
+            throw new ParseException(String.format(MESSAGE_INVALID_HOUR));
+        }
+
+        if (hour <= 0) {
+            throw new ParseException(String.format(MESSAGE_INVALID_HOUR));
+        }
+
```

Comment:
Should we specify more things as a valid hour? So that no further clash can happen
Example: a lesson must be a multiple of 0.5 hour, so no such lesson last 45 minutes -> we can also solve the pay amount and owe amount problem as a 2 decimal places number multiplies by 0.5 never produces more decimal places?

Reply:
Im not very sure about this as this might be too restrictive for the user since over-running a lesson by 40 mins, 15 mins etc is possible. What do yall think? @teamotitan @Feng1231 @Soumyadip-cmd